props/blaster.h

```diff
@@ -301,22 +303,32 @@ class Blaster : public PropBase {
       case EFFECT_CLIP_IN: hybrid_font.PlayCommon(&SFX_clipin); return;
       case EFFECT_CLIP_OUT: hybrid_font.PlayCommon(&SFX_clipout); return;
       case EFFECT_RELOAD: hybrid_font.PlayCommon(&SFX_reload); return;
-      case EFFECT_MODE:
-	if (SFX_mode) {
-	  hybrid_font.PlayCommon(&SFX_mode);
-	  return;
-	}
-	// TODO: would rather do a Talkie to speak the mode we're in after mode sound
-	beeper.Beep(0.05, 2000.0);
-	return;
+      case EFFECT_MODE: SayMode(); return;
       case EFFECT_RANGE: hybrid_font.PlayCommon(&SFX_range); return;
       case EFFECT_EMPTY: hybrid_font.PlayCommon(&SFX_empty); return;
       case EFFECT_FULL: hybrid_font.PlayCommon(&SFX_full); return;
       case EFFECT_JAM: hybrid_font.PlayCommon(&SFX_jam); return;
       case EFFECT_UNJAM: hybrid_font.PlayCommon(&SFX_unjam); return;
       case EFFECT_PLI_ON: hybrid_font.PlayCommon(&SFX_plion); return;
       case EFFECT_PLI_OFF: hybrid_font.PlayCommon(&SFX_plioff); return;
-	
+  
+    }
+  }
+
+  void SayMode() {
+    switch(blaster_mode) {
+      case MODE_STUN:
+      if (SFX_mdstun) hybrid_font.PlayCommon(&SFX_mdstun); return;
```

Comment:
Did you test it?
(Hint; thjis doesn't work.)

Reply:
I was dumb and lazy and just commented out the font files checks to avoid a song and dance to delete the files.
That being said, it's corrected and tested properly now.
Why wouldn't it fall through to the talkie if no sounds were found? I was just trying to condense the if clause to one line for neatness/efficiency.